go.mod

```diff
@@ -13,59 +13,59 @@ require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/zipkin v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/attributesprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/groupbyattrsprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstransformprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/probabilisticsamplerprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/cloudfoundryreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/collectdreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/journaldreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8seventsreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkametricsreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kubeletstatsreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mongodbatlasreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusexecreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/receivercreator v0.54.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sapmreceiver v0.54.0
```

Comment:
I believe we need https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11518 as well